lib/requestwrapper.ts

```diff
@@ -14,8 +14,8 @@
  * limitations under the License.
  */
 
-import * as extend from 'extend';
```

Comment:
```
import extend from 'extend';
```

Reply:
That doesn't work. The type definition for `extend` is written in such a way that it overloads a bunch of functions. The alternative to `import *` is `import extend = require('extend')`